pygpu/tests/test_elemwise.py

```diff
@@ -101,87 +100,6 @@ def ielemwise2_ops_array(op, dtype1, dtype2, shape):
     assert numpy.allclose(out_c, numpy.asarray(out_g), atol=1e-6)
 
 
-def test_elemwise_layouts():
```

Comment:
Why remove those tests? I think this tests more cases then the C tests. So I would have a tendency to keep them.